packages/compass-e2e-tests/helpers/commands/add-collection.ts

```diff
@@ -68,20 +68,38 @@ export async function addCollection(
     for (const [key, value] of Object.entries(
       collectionOptions.customCollation
     )) {
+      const valStr = value.toString();
+
       await browser.clickVisible(
         Selectors.createCollectionCustomCollationFieldButton(key)
       );
-      const menu = await browser.$(
+      const menu = browser.$(
         Selectors.createCollectionCustomCollationFieldMenu(key)
       );
       await menu.waitForDisplayed();
-      const span = await menu.$(`span=${value.toString()}`);
-      await span.waitForDisplayed();
-      await span.scrollIntoView();
-      await browser.screenshot(
-        `custom-collation-${key}-${value.toString()}.png`
+      const option = menu.$(`li[value="${valStr}"]`);
+
+      let clickAttempt = 1;
+
+      // Just continue clicking until the value is selected ...
+      await browser.waitUntil(
+        async () => {
+          await browser.clickVisible(option, {
+            scroll: true,
+            screenshot: `custom-collation-${key}-${valStr}-${clickAttempt}.png`,
```

Comment:
This is easily going to upload 1000s of screenshots 🙈

Reply:
Ah, let me remove that actually, was helpful for debugging, but definitely an overkill for the real thing (I'll just do one at the end)

Reply:
Removed in https://github.com/mongodb-js/compass/pull/5400/commits/223e0e6be8966983c7cfef7cbb0e47dd6bd12980